packages/app-builder-lib/src/publish/BitbucketPublisher.ts

```diff
@@ -18,11 +18,13 @@ export class BitbucketPublisher extends HttpPublisher {
     super(context)
 
     const token = process.env.BITBUCKET_TOKEN
+    const username = process.env.BITBUCKET_USERNAME
+
```

Comment:
Can you add some warning logging so that this usage is evident that we're defaulting to the user repo?
```
    if (isEmptyOrSpaces(username)) {
      log.warn('No Bitbucket username provided via "BITBUCKET_USERNAME". Defaulting to use repo owner.')
    }
```